src/Field.tsx

```diff
@@ -81,7 +81,7 @@ export function useField<Val = any>(
   }
 
   if (isObject(propsOrFieldName)) {
-    if (process.env.NODE_ENV !== 'production') {
+    if (__DEV__) {
```

Comment:
I'm not sure which way is better: to use `process.env.NODE_ENV !== 'production'` or `__DEV__`, but I think we need to keep only one of them. I prefer `__DEV__`

Reply:
Use __DEV__